### PR TITLE
Update style–guides

### PR DESCRIPTION
**Pull request by arkhi**
1. Rename files to group them by category.
1. Move files from documentation repository:
  * Add HTML and CSS style-guides.
  * Refer original documentation for PHP PSR-2.
  * Fix typo in “Beautiful Computer Programming”.
1. Give more details about our implementation of Idiomatic style–guides:
  * Indentation for attributes and closing bracket when writing elements on multiple lines.
  * Mention `'use strict'`.
  * Add .jscsrc and .jshintrc to repository.

### CODE REVIEW

File: workflow-beautiful-code-programming.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Beatiful Computer Programming
+# Beautiful Computer Programming
```
Comment (imlazyone) [inline workflow-beautiful-code-programming.md]: :laughing:

**Comment by imlazyone** [comment]
All good for me, @juwai/frontend please review

**Comment by arkhi** [comment]
@juwai/all: I’ll merge on Thursday if no one has comments until then.

**Comment by wenzong** [comment]
:+1:

**Comment by diartyz** [comment]
is there a way to link it in other project ?

**Comment by HouCoder** [comment]
:+1:

**Comment by arkhi** [comment]
I merged on Wednesday in the end… It’s been here for too long already. :)

Anybody who want to amend anything can make a pull–request.

**Comment by arkhi** [comment]
Closes juwai/in-house-tasks/issues/16.

**Pull request merged**